### PR TITLE
Tweak formatting to prepare for switching to black

### PR DESCRIPTION
**Pull request by maffoo**
Switching the formatting to black in #3516 brought up a few issues. Black has an edge case that requires inserting trailing commas in some constructs (https://github.com/psf/black/issues/1629), which I've done here. Also removes trailing whitespace in a few places that pylint complains about when the files are reformatted (those lint errors are latent since we only lint changed files on PRs).

### CODE REVIEW

File: cirq/ops/pauli_string_test.py [hunk 1]
```diff
@@ -1676,13 +1677,18 @@ def test_circuit_diagram_info():
             1j * cirq.X(a) * cirq.Y(b),
             -1j * cirq.Y(b),
             1j**0.5 * cirq.X(a) * cirq.Y(b),
-        ), """
+        ),
+        """
```
Comment (maffoo) [inline cirq/ops/pauli_string_test.py:1681]: The trailing comma causes yapf to move the leading quotes to their own line, which for some reason seems to confuse pylint. I could only get it to ignore this long string literal by putting the `pylint: disable` directive at the top level. No idea why.

File: cirq/contrib/qasm_import/_parser_test.py [hunk 2]
```diff
@@ -67,12 +67,12 @@ def test_comments():
     parser = QasmParser()
 
     parsed_qasm = parser.parse("""
-    //this is the format 
+    //this is the format
```
Comment (viathor) [inline cirq/contrib/qasm_import/_parser_test.py:70]: nit: add whitespace between // and text for consistency

File: cirq/interop/quirk/url_to_circuit_test.py [hunk 3]
```diff
@@ -79,15 +79,15 @@ def test_parse_with_qubits():
 
     assert quirk_url_to_circuit(
         'http://algassert.com/quirk#circuit={"cols":[["H"],["•","X"]]}',
-        qubits=cirq.GridQubit.rect(4, 4)) == cirq.Circuit(
-            cirq.H(a),
-            cirq.X(b).controlled_by(a))
+        qubits=cirq.GridQubit.rect(4, 4),
+    ) == cirq.Circuit(cirq.H(a),
```
Comment (viathor) [inline cirq/interop/quirk/url_to_circuit_test.py:83]: It's fairly common to add commas in long literal lists/tuples/sets/etc, but I suppose people are likely to forget to do this in function argument lists. From a quick glance at https://github.com/psf/black/issues/1629, it seems that the fallout is fairly scary: black produces inequivalent code! However, they do have a workaround (running twice with --fast), so perhaps we should use that in #3516? Added a comment there.

File: cirq/ops/pauli_string_test.py (hunk 1 above)
Comment (balopat) [reply cirq/ops/pauli_string_test.py:1681]: Hmm, the battle of the "productivity" tools...well, soon enough yapf is going to go.

**Review by balopat — APPROVED** [review]
LGTM

**Review by viathor — APPROVED** [review]

**Review by HGSilveri — COMMENTED** [review]
LGTM

**Pull request merged**